core/integration/bookkeeping/plugin.go

```diff
@@ -234,9 +236,52 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 		}
 		return
 	}
-	updateRunFunc := func(runNumber64 int64, state string) (out string) {
+	updateRunFunc := func(runNumber64 int64, state string, timeO2Start time.Time, timeO2End time.Time, timeTrgStart time.Time, timeTrgEnd time.Time) (out string) {
 		callFailedStr := "Bookkeeping UpdateRun call failed"
-		err := p.bookkeepingClient.UpdateRun(int32(runNumber64), state, time.Now(), time.Now())
+		//odc_topology_fullname, _ := env.Workflow().GetVars().Get("odc_topology_fullname")
+		trg_global_run_enabled, err := strconv.ParseBool(env.GetKV("", "trg_global_run_enabled"))
+		if err != nil {
```

Comment:
In this group of parses if a single parse fails the update function exits; wouldn't we want to update even if some field is missing?

Reply:
I was wondering about this, but I guess you're right, I will change this to not exit on parse error.